integration/dbt/README.md

```diff
@@ -14,7 +14,7 @@ Wrapper script for dbt for automatic metadata collection
 - [Python >= 3.6](https://www.python.org/downloads)
 - [dbt >= 0.20](https://www.getdbt.com/)
 
-Right now, `openlineage-dbt` only supports `bigquery`, `snowflake` and `redshift` dbt adapters.
+Right now, `openlineage-dbt` only supports `bigquery`, `snowflake`, `spark` and `redshift` dbt adapters.
```

Comment:
Maybe we should switch to a table format to list adapters? With information on:

* Version supported
* Future adopter support
* Etc.

Reply:
Good idea. There's a lot of possible adapters, and this will make implementing future ones much clearer.